app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -372,6 +372,10 @@ class DefaultSessionControlController(
             TopSite.Type.PINNED -> metrics.track(Event.TopSiteOpenPinned)
         }
 
+        if (url == SupportUtils.GOOGLE_URL) {
```

Comment:
Just to confirm - the tile always has GOOGLE_URL and then after the click we determine where it should actually go (GOOGLE_US, GOOGLE_XX) so at this point in thime `url` is really always `GOOGLE_URL` ?

Reply:
Yes

Reply:
You will see `url = appendSearchAttributionToUrlIfNeeded(url),` on line 384 below.